src/main/java/org/apache/sling/resourceresolver/impl/ResourceResolverFactoryConfig.java

```diff
@@ -163,15 +163,27 @@
     @AttributeDefinition(name = "Allowed Vanity Path Location",
         description ="This setting can contain a list of path prefixes, e.g. /libs/, /content/. If " +
                     "such a list is configured, only vanity paths from resources starting with this prefix " +
-                    " are considered. If the list is empty, all vanity paths are used.")
+                    " are considered. If the list is empty, we fallback to resource_resolver_vanitypath_allowedlist.")
     String[] resource_resolver_vanitypath_whitelist();
 
+    @AttributeDefinition(name = "Allowed Vanity Path Location",
+        description ="This setting can contain a list of path prefixes, e.g. /libs/, /content/. If " +
+            "such a list is configured, only vanity paths from resources starting with this prefix " +
+            " are considered. If the list is empty, all vanity paths are used.")
+    String[] resource_resolver_vanitypath_allowedlist();
```

Comment:
Please use 'allowlist' instead of 'allowedlist' . See also https://cwiki.apache.org/confluence/display/SLING/Removal+of+problematic+language